demo/vl-select.html

```diff
@@ -313,6 +314,31 @@
             </select>
             <button is="vl-button" onclick="document.getElementById('select-dressed-focus').focus();" slot="actions">Focus</button>
         </vl-demo>
+
+        <vl-demo data-vl-title="Required dressed">
+          <form is="vl-form" id="select-dressed-required-form">
+            <select id="select-dressed-required" is="vl-select" data-vl-block data-vl-error-placeholder="select-dressed-required-error" data-vl-required="true"></select>
+          </form>
+
+          <div slot="actions">
+            <button id="activate-data-button-required-dressed" is="vl-button" onclick="laadDynamischDressedRequired()" type="button">Activeer dynamische data</button>
+          </div>
+
+          <script type="text/javascript">
+                laadDynamischDressedRequired = () => {
+                  document.getElementById('select-dressed-required').dress({
+                    callbackFn: (select) => {
+                      return [
+                        {value: 'one', label: 'Label one'},
+                        {value: 'two', label: 'Label two'},
+                        {value: 'three', label: 'Label three'},
+                      ];
+                    },
+                  });
+                };
+          </script>
+        </vl-demo>
```

Comment:
De demo is overbodig op e2e niveau, aangezien dit voorbeeld voor afnemers geen meerwaarde vormt t.o.v. de andere demo's.

Reply:
omgevormd naar unit test